README.md

```diff
@@ -27,90 +27,53 @@ Convenience commands have been added to `fabfile.py` to help you interact with t
 Once installed, you can type `fab -l` to see a list of available commands.
 
 
-### 1. Build Docker containers
+### 1. Build and start Docker containers
 
-```console
-$ fab build
-```
-
-### 2. Start Docker containers
-
-```console
-$ fab start
-```
-
-### 3. Start a shell session with the 'web' container
-
-```console
-$ fab sh
-```
-
-### 5. Apply database migrations
-
-```console
-$ python manage.py migrate
+```sh
+fab start
 ```
 
-### 6. Run the 'development' web server
+This command takes care of the following:
 
-```console
-$ python manage.py runserver 0.0.0.0:8000
-```
+1. Building all of the necessary Docker containers
+1. Starting all of the necessary Docker containers
+2. Installing any new python dependencies
+3. Applying any new database migrations
+4. Starting the Django development server
 
-### 7. Access the site
+### 2. Access the site
 
 <http://127.0.0.1:8000>
 
-**Note: compiled CSS is not included and therefore needs to be built initially, and after each git pull. See the "Working with SASS/CSS section**
+### 3. Create a Django user for yourself
 
-### 8. Create a Django user for yourself
-
-```console
-$ python manage.py createsuperuser
+```sh
+python manage.py createsuperuser
```

Comment:
I think we still need to do `fab sh` to get into the container to run these commands

Reply:
Yes we do. Will review the readme...

Reply:
Updated. Does the README look a little better now?